app/controllers/allies_controller.rb

```diff
@@ -109,15 +109,4 @@ def remove
       format.json { head :no_content }
     end
   end
-
-  private
-
-  def if_not_signed_in
```

Comment:
Check the [`application_controller.rb:46`](https://github.com/julianguyen/ifme/blob/master/app/controllers/application_controller.rb#L46) there is a duplicated method!